.devcontainer/scripts/requirements.txt

```diff
@@ -0,0 +1,30 @@
+astroid==2.15.6
+build==0.10.0
+certifi==2023.5.7
+charset-normalizer==3.2.0
+colorama==0.4.6
+dill==0.3.6
+docker==6.1.3
+future-fstrings==1.2.0
+idna==3.4
+iniconfig==2.0.0
+isort==5.12.0
+lazy-object-proxy==1.9.0
+mccabe==0.7.0
+networkx==3.1
+packaging==23.1
+platformdirs==3.8.1
+pluggy==1.2.0
+pylint==2.17.4
+pyproject_hooks==1.0.0
+pytest==7.4.0
+pytest-depends==1.0.1
+PyYAML==6.0
+requests==2.31.0
+tomlkit==0.11.8
+urllib3==2.0.3
+websocket-client==1.6.1
+wrapt==1.15.0
+robotframework==7.0.0
+typing_extensions==4.9.0
+skyramp
```

Comment:
I think I made this change a while back - did you want to undo it, or is it fine?

Reply:
since we are using the latest binary, let's use the latest python library. Thanks for making this change